sebsync.py

```diff
@@ -126,54 +131,89 @@ def get_remote_ebooks() -> None:
             id=entry.find("dc:identifier", ns).text,
             title=entry.find("atom:title", ns).text,
             author=entry.find("atom:author", ns).find("atom:name", ns).text,
-            href=entry.find(f".//atom:link[@title='{type_selector[options.type]}']", ns).attrib[
-                "href"
-            ],
+            href=entry.find(
```

Comment:
This change (and others) seem like pure formatting. I'm guessing you're using black (or your editor is performing its own reformatting.) This project's currently using ruff for formatting, based on the configuration in `pyproject.toml`. Running `uv run ruff format` should undo such changes.

Reply:
Yeah, I run black out of habit. I'll take a look at ruff.